packages/vite-plugin-svelte/src/ui/inspector/plugin.ts

```diff
@@ -84,32 +76,10 @@ export function svelteInspector(): Plugin {
 			}
 		},
 
-		transform(code: string, id: string, options?: { ssr?: boolean }) {
-			if (options?.ssr || disabled || !appendTo) {
-				return;
+		transform(code: string, id: string) {
+			if (id.includes('vite/dist/client/client.mjs')) {
+				return { code: `${code}\nimport('virtual:svelte-inspector-path:load-inspector.js')` };
```

Comment:
Dynamic import since the `Inspector.svelte` would rely on `/@vite/client` too, and it otherwise would cause an import loop an runtime `ReferenceError`.